www/js/plugin/storage.ts

```diff
@@ -25,7 +25,9 @@ const unmungeValue = (key, retData) => {
 }
 
 const localStorageSet = (key: string, value: {[k: string]: any}) => {
-  localStorage.setItem(key, JSON.stringify(value));
+  if (value) {
+    localStorage.setItem(key, JSON.stringify(value));
+  }
```

Comment:
would you add a comment and a link to the discussion around this here for clarity?

Reply:
done in [b2d6aa9](https://github.com/e-mission/e-mission-phone/pull/1072/commits/b2d6aa912f152515d791f136654b696acff125d1)